utility/mm2_propagation

```diff
@@ -166,42 +251,59 @@ fd.close()
 ind = np.arange(len(labels))
 width = 0.62
 
-fig = plt.figure(figsize=(16, 8))
+fig = plt.figure(figsize=(14, 7))
 ax = fig.add_subplot(111)
 
 rects = ax.bar(ind, repomd_same, width, color='red', label="synced")
 label_bars_size(rects, '%.0f')
 
 rects = ax.bar(ind, repomd_one_day, width, bottom=repomd_same,
-        color='mediumvioletred',
-        label='synced - 1')
+               color='mediumvioletred',
+               label='synced - 1')
 label_bars_size(rects, '%.0f', repomd_same)
 
 bottom = list_adder(repomd_same, repomd_one_day)
 rects = ax.bar(ind, repomd_two_day, width, bottom=bottom, color='b',
-        label='synced - 2')
+               label='synced - 2')
 label_bars_size(rects, '%.0f', bottom)
 
 bottom = list_adder(bottom, repomd_two_day)
-rects = ax.bar(ind, repomd_other, width, bottom=bottom, color='g', label='older')
+rects = ax.bar(
+    ind,
+    repomd_other,
+    width,
+    bottom=bottom,
+    color='g',
+    label='older')
 label_bars_size(rects, '%.0f', bottom)
 
 bottom = list_adder(bottom, repomd_other)
 rects = ax.bar(ind, repomd_none, width, bottom=bottom, color='y', label='N/A')
-label_bars_size(rects, '%.0f', bottom)
+label_bars_size(rects, '%.0f', bottom, True)
 
-ax.set_ylim(0, maximum +40)
-ax.set_xticks(ind+width)
+
+ax.set_ylim(0, maximum + 40)
+ax.set_xticks(ind + width)
 ax.set_xticklabels(labels)
 plt.setp(plt.xticks()[1], rotation=90)
 
 plt.legend(loc=2, shadow=True, fancybox=True, ncol=3)
-if prefix == 'f22-updates-':
-    plt.title("repomd.xml propagation for updates/22/x86 64/repodata")
-else:
-    plt.title("repomd.xml propagation for development/rawhide/x86 64/os/repodata")
-plt.savefig("%s%s-repomd-propagation.pdf" % (prefix,labels[len(labels)-1].replace('\n','-')), bbox_inches='tight')
-plt.savefig("%s%s-repomd-propagation.svg" % (prefix,labels[len(labels)-1].replace('\n','-')), bbox_inches='tight')
-print "%s%s-repomd-propagation.pdf" % (prefix,labels[len(labels)-1].replace('\n','-'))
-plt.savefig("%srepomd-propagation.pdf" % prefix , bbox_inches='tight')
-plt.savefig("%srepomd-propagation.svg" % prefix , bbox_inches='tight')
+
+
+plt.title("repomd.xml propagation for %s" % proppath)
+
+plt.savefig(
+    "%-s-%s-repomd-propagation.pdf" %
+    (args.prefix, labels[len(labels) - 1].split('\n')[0]),
+    bbox_inches='tight')
+
+plt.savefig(
+    "%s-%s-repomd-propagation.svg" %
+    (args.prefix, labels[len(labels) - 1].split('\n')[0]),
+    bbox_inches='tight')
+
+print("%s-%s-repomd-propagation.pdf" %
```

Comment:
This is meant to be run in a cron job right? If so, should we only print when debugging?